### DISCUSSION TITLE
Configuration of attachment server endpoints on digital ocean

### DISCUSSION THREAD

**Discussion by oxsnard** [mastodon/mastodon#21720] [Q&A]
Hi all, im working on a project that involves a small instance deployed on a digital ocean droplet. I used the DO spaces feature and images uploaded to the server via the web app are showing up in the correct spaces URL see here:

https://tootberg.sfo3.digitaloceanspaces.com/media_attachments/files/109/414/480/342/385/273/small/d188594c4d40dcb7.png

however, the web client is trying to display an image posted here:

https://files.tootberg.xyz/media_attachments/files/109/414/480/342/385/273/small/d188594c4d40dcb7.png

This one gives a 404

The syntax is correct and consistent, meaning that the client is interfacing with the DO spaces service as expected on the upload side but not on the display side. Do i need to modify a setting in the droplet via console?

Thanks in advance

Edit: I'm pretty sure my CORS headers are fine in Digital Ocean, since uploads are enabled from the web UI

**Comment by davetroy** [comment]
What's your Nginx config look like for `files.tootberg.xyz`? You should have that setup as a proxy cache in front of your S3 bucket.
